packages/rps/src/redux/message-service/firebase-inbox-listener.ts

```diff
@@ -10,7 +10,8 @@ export function* firebaseInboxListener(client: RPSChannelClient) {
     reduxSagaFirebase.database.channel,
     `/messages/${address.toLowerCase()}`,
     'child_added',
-    buffers.fixed(10)
+    // TODO: @george why was buffers added here?
+    // buffers.fixed(10)
```

Comment:
`call` doesn't have an argument for this at the end, so I am a bit confused. It was throwing an error after upgrading TypeScript. @geoknee

Reply:
For context: You also updated `redux-saga`.

I'm not sure why this is here, though. Looks like it was probably my mistake: perhaps I saw `channel` on L10 and read `actionChannel` https://redux-saga.js.org/docs/api/#actionchannelpattern-buffer which does take a buffer argument.

Reply:
I think you're good to just remove it.